backend/src/main/java/com/bakdata/conquery/models/query/DistributedExecutionManager.java

```diff
@@ -177,17 +207,19 @@ public void clearQueryResults(ManagedExecution execution) {
 
 	@Override
 	public Stream<EntityResult> streamQueryResults(ManagedExecution execution) {
-		final List<List<EntityResult>> resultParts = executionResults.getIfPresent(execution.getId());
+		final Map<?, List<EntityResult>> resultParts = executionResults.getIfPresent(execution.getId());
```

Comment:
was ist das ?, oder ist das egal?

Reply:
das ist die workerId, die ist an der Stelle egal.

https://github.com/ingef/conquery/pull/3347/commits/c3a3055c2ccc7b4a78b4f9128e5676540c155922

in dem commit glaube nochmal klarer was passiert